src/H4LTools.cc

```diff
@@ -948,7 +951,15 @@ bool H4LTools::ZZSelection_2l2q(){
 
         if (FatJetidx.size() > 0)
         {
+
             Z2.SetPtEtaPhiM(FatJet_pt[0], FatJet_eta[0], FatJet_phi[0], FatJet_SDmass[0]);
+
+            TLorentzVector Z2_1;//AV
+            TLorentzVector Z2_2;//AV
+            Z2_1.SetPtEtaPhiM(Jet_pt[0], Jet_eta[0], Jet_phi[0], Jet_mass[0]);
+            Z2_2.SetPtEtaPhiM(Jet_pt[1], Jet_eta[1], Jet_phi[1], Jet_mass[1]);
+            Z2_2j = Z2_1 + Z2_2;
```

Comment:
I think here, we will have some issue. 

Suppose in an event if there are less than two AK4 jets then this will give us error.

So, we need to protect this and skip the error if it appears.